packages/design/src/ResponsiveBreakpoint.stories.mdx

```diff
@@ -20,11 +20,11 @@ the @media breakpoint in your CSS.
 
 | Breakpoint                     | Width      |
 | :----------------------------- | :--------- |
-| `--small-screens-and-below`    | `<490px`   |
-| `--small-screens-and-up`       | `>=490px`  |
-| `--medium-screens-and-up`      | `>=768px`  |
-| `--large-screens-and-up`       | `>=1440px` |
-| `--extra-large-screens-and-up` | `>=1680px` |
+| `--small-screens-and-below`    | `< 490px`  |
+| `--small-screens-and-up`       | `>= 490px` |
+| `--medium-screens-and-up`      | `>= 768px` |
+| `--large-screens-and-up`       | `>= 1080`  |
+| `--extra-large-screens-and-up` | `>= 1440`  |
```

Comment:
@darryltec is there a specific reason these two aren't labelled with `px`?

Reply:
nope. totally forgot about it